maestro/deb/build.sh

```diff
@@ -4,10 +4,8 @@
 PELION_PACKAGE_NAME="maestro"
 PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
 
-PELION_PACKAGE_PRE_BUILD_CALLBACK='select_python 2'
-
 declare -A PELION_PACKAGE_COMPONENTS=(
-    ["https://github.com/armPelionEdge/maestro.git"]="d6f1e29c2994c34adcf544d0fff50fd31fd01cdb")
+    ["https://github.com/armPelionEdge/maestro.git"]="718a61ce09dabc60075322e1c255c345aa7927e6")
```

Comment:
Domain is wrong, needs to be updated to PelionIoT.

Reply:
I'd prefer to leave this PR for taking the changes from master => dev.
There are 38 places in the dev branch where the domain name needs to be updated, so those can be done in separate PR.

Reply:
Roger that.